### PR TITLE
apply DAO updates DAO-04M

### PR DESCRIPTION
**Pull request by shalaquiana**

### CODE REVIEW

File: contracts/DAO.sol [hunk 1]
```diff
@@ -41,7 +41,10 @@ contract DAO {
     uint256 public trg;//total rewards generated
     mapping(address=>uint256) public lrc;//last rewards claimed at trg point for this address
     mapping(address=>uint256) public lST;//last stake time
-
+    mapping(address=>uint256) public lastClaimPoint;//point at which above a user is eligable to claim
+    mapping(uint256=>uint256) public claimPoints;//points of ETH transfers to this contract
+    mapping(uint256=>uint256) public totalStakedAtClaimPoint;//points of ETH transfers to this contract
```
Comment (calikevuche) [inline contracts/DAO.sol:46]: Are duplicate comments intentional?

File: contracts/DAO.sol [hunk 2]
```diff
@@ -138,26 +140,36 @@ contract DAO {
...
+            
+        currentClaimPoint = currentClaimPoint+1;
+        totalStakedAtClaimPoint[currentClaimPoint] = totalStaked();
+    }
+
     /**
     * @notice base ETH rewards since last claim
     * @param acc the account to get the answer for
     */
     function bRSLC(address acc) public view returns (uint256) {
```
Comment (calikevuche) [inline contracts/DAO.sol:156]: maybe we should use `account` instead of `acc` to be consistent with other functions.

**Review by calikevuche — APPROVED** [review]
benign comments, otherwise LGTM

File: contracts/DAO.sol (hunk 1 above)
Comment (shalaquiana) [reply contracts/DAO.sol]: good point I've added a accurate comment for totalStakedAtClaimPoint

File: contracts/DAO.sol (hunk 2 above)
Comment (shalaquiana) [reply contracts/DAO.sol]: nice catch, have done so

**Pull request merged**